### PR TITLE
Adds openshift_fs_inotify_max_user_instances to the node tuned profile.

### PR DESCRIPTION
**Pull request by jtudelag**
Work around for: https://github.com/kubernetes/kubernetes/issues/64315

BZ: https://bugzilla.redhat.com/show_bug.cgi?id=1559061

### CODE REVIEW

**Comment by jtudelag** [comment]
Not sure if this should be included here instead, as a default: https://github.com/openshift/openshift-ansible/blob/master/roles/tuned/templates/openshift/tuned.conf

fs.inotify.max_user_watches is set twice: 

* https://github.com/openshift/openshift-ansible/blob/master/roles/tuned/templates/openshift/tuned.conf#L18

* https://github.com/openshift/openshift-ansible/blob/master/roles/tuned/templates/openshift-node/tuned.conf#L11

**Comment by vrutkovs** [comment]
bot, test pull request

**Comment by sdodson** [comment]
@sjenning Is this still an appropriate solution to the referenced bug or will this be fixed in the kubelet?

**Comment by aveshagarwal** [comment]
> @sjenning Is this still an appropriate solution to the referenced bug or will this be fixed in the kubelet?

My understanding is that based on the comment: https://bugzilla.redhat.com/show_bug.cgi?id=1559061#c3, it looks like  is a similar to https://bugzilla.redhat.com/show_bug.cgi?id=1559061 https://bugzilla.redhat.com/show_bug.cgi?id=1589656.

In later kube releases, the issue was fixed in https://github.com/google/cadvisor/pull/1916 but later reverted here: https://github.com/kubernetes/kubernetes/pull/64987. Even if this fix is later added, it is too large to backport to older releases like 3.6. 

So in summary, I'd say, having a workaround like this PR should be ok.

**Comment by sdodson** [comment]
@aveshagarwal thanks, we'll work on backporting it.

**Comment by wmengRH** [comment]
@vrutkovs May I know  just backport this to 3.10 or more versions? Thanks.
https://github.com/openshift/openshift-ansible/pull/9204#issuecomment-405191007
